sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogV2Implicits.scala

```diff
@@ -132,4 +132,9 @@ private[sql] object CatalogV2Implicits {
       part
     }
   }
+
+  def unqote(input: String): Seq[String] = {
+    // TODO: Handle the column name contains `dot` properly
+    input.split('.')
```

Comment:
You can copy how this is done in [`LogicalExpressions`](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/connector/expressions/expressions.scala#L32-L34).

```scala
val parser = new CatalystSqlParser(SQLConf.get)
val nameParts = parser.parseMultipartIdentifier(name)
```

You can also use the `parseReference` method if you want to get a `FieldReference`.

Reply:
Thanks for the tip. I was planing to implement this myself.